docs/integration/alerts-squadcast-integration.md

```diff
@@ -0,0 +1,54 @@
+
+**3.** If you don't have an API token, click the **Generate new API token** to generate a token used to communicate with Squadcast and copy the it so you can use it in Sematext:
+
+<img class="content-modal-image" alt="Create Squadcast Integration - New API Token" src="../../images/integrations/create-squadcast-integration-profile-details.png" title="Create Squadcast Integration - New API Token">
+
+**4.** If you don't have a service, create it using the **Add Service** button in the **Services** section. If you already have a service defined and you would like to add **alert source** jump to point **6**:
+
+<img class="content-modal-image" alt="Create Squadcast Integration - Add Service" src="../../images/integrations/create-squadcast-integration-add-service.png" title="Create Squadcast Integration - Add Service">
+
+**5.** A modal window will open allowing you to define the service
+
+<img class="content-modal-image" alt="Create Squadcast Integration - Service Details" src="../../images/integrations/create-squadcast-integration-service-details.png" title="Create Squadcast Integration - Service Details">
+
+**6.** In **Services** click the **Alert Sources** under the service you would like to integrate Sematext notifications with. If you wish to receive notification from Sematext to different services you will need to create multiple notification hooks:  
+
+<img class="content-modal-image" alt="Create Squadcast Integration - Alert Sources" src="../../images/integrations/create-squadcast-integration-alert-sources.png" title="Create Squadcast Integration - Alert Sources">
+
+**6.** In the opened window type **API** and copy the **API Webhook URL**:
+
+<img class="content-modal-image" alt="Create Squadcast Integration - API Webhook URL" src="../../images/integrations/create-squadcast-integration-alert-source-details.png" title="Create Squadcast Integration - API Webhook URL">
+
+Having the **Alert Source** configured and having the **API Webhook URL** we can add the notification to Sematext. 
+
+## In Sematext
+
+**1.** Navigate to [Notification Hooks](https://apps.sematext.com/ui/webhook-create) and select Squadcast card to create a new Squadcast notification hook.
+
+![Sematext Notification Hooks](../../images/integrations/sematext-notification-hooks.png "Sematext Notification Hook")
+
+**2.** Add your Squadcast **Webhook URL**. 
+
+<img class="content-modal-image" alt="Create Squadcast Integration" src="../../images/integrations/create-squadcast-integration.png" title="Create Squadcast Integration">
+
+Next, click the **Send Test Notification** button. Squadcast returns information similar to **{"data":{"message":"Pipeline processing done"},"meta":{"request_id":"8a8c2d9b-aea6-479b-9f1f-373ac6785d37"}}** indicating everything is configured correctly. Check your Squadcast integration for the test message from Sematext. 
```

Comment:
returns => should return
(this may be in other pages, so please change there, too)

Reply:
Will do, thanks.